src/background/background.ts

```
@@ -0,0 +1,295 @@
+
+    switch (command) {
+      case '_execute_action':
+        // Show snippet picker
+        await this.browser.tabs.sendMessage(tab.id, {
+          type: 'showSnippetPicker'
+        });
+        break;
+
+      case 'toggle_expansion':
+        // Toggle expansion on/off
+        const settings = await this.storageManager.getSettings();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)